src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1892,6 +1893,38 @@ impl Coordinator {
         };
     }
 
+    pub(super) async fn sequence_explain_pushdown(
+        &mut self,
+        ctx: ExecuteContext,
+        plan: plan::ExplainPushdownPlan,
+        target_cluster: TargetCluster,
+    ) {
+        match plan.explainee {
+            Explainee::Statement(ExplaineeStatement::Select {
+                broken: false,
+                plan,
+                desc: _,
+            }) => {
+                self.execute_peek_stage(
+                    ctx,
+                    OpenTelemetryContext::obtain(),
+                    PeekStage::Validate(PeekStageValidate {
+                        plan,
+                        target_cluster,
+                        copy_to_ctx: None,
+                        explain_ctx: ExplainContext::Pushdown,
+                    }),
+                )
+                .await;
+            }
+            _ => {
+                ctx.retire(Err(AdapterError::Unsupported(
+                    "EXPLAIN FILTER PUSHDOWN queries for this query type",
```

Comment:
To use consistent wording with the rest of our code:

```suggestion
                    "EXPLAIN FILTER PUSHDOWN queries for this explainee type",
```